sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,25 +1,45 @@
 # Release History
 
-## 1.8.0-beta.4 (Unreleased)
+## 1.8.0 (2023-02-02)
 
 ### Features Added
 
+- Added support for parsing space character in place of 'T' in RFC3339 DateTimes.
```

Comment:
- Update OpenTelemetry attributes to align with current Azure guidelines for OpenTelemetry attributes and names.